nuts/client/auth.go

```diff
@@ -66,6 +69,32 @@ func (c HTTPClient) GetIrmaSessionResult(sessionToken string) ([]byte, error) {
 	return testAndReadResponse(http.StatusOK, resp)
 }
 
+func (c HTTPClient) CreateDummySession(customer domain.Customer) ([]byte, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	body := nutsAuthClient.CreateSignSessionJSONRequestBody{
+		Means:   "dummy",
+		Payload: "Ik verklaar te handelen namens " + customer.Name,
+	}
+
+	resp, err := c.auth().CreateSignSession(ctx, body)
+	if err != nil {
+		return nil, err
+	}
+	return testAndReadResponse(http.StatusCreated, resp)
+}
+
+func (c HTTPClient) GetDummySessionResult(sessionToken string) ([]byte, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+	resp, err := c.auth().GetSignSessionStatus(ctx, sessionToken)
```

Comment:
this must be done twice before you get a VP?

Reply:
What do you mean by twice?

Reply:
https://github.com/nuts-foundation/nuts-node/blob/master/auth/services/dummy/dummy.go#L188

status is not completed until the third call